android/src/main/java/com/ioreactnativecieid/IoReactNativeCieidModule.kt

```diff
@@ -51,6 +84,11 @@ class IoReactNativeCieidModule(reactContext: ReactApplicationContext) :
       }
 
       try {
+        if(!isSignatureValid(packageName, signature)) {
+          onActivityResultCallback = null
+          ME.CIEID_SIGNATURE_MISMATCH.invoke(resultCallback)
+          return;
```

Comment:
```suggestion
          return
```
We don't need that "old" terminator :P

Reply:
Good to know! Thanks! 👀